pyvista/plotting/themes.py

```diff
@@ -2822,7 +2841,7 @@ def load_theme(self, theme):
                 '``theme`` must be a pyvista theme like ``pyvista.plotting.themes.Theme``.'
             )
 
-        for attr_name in theme.__slots__:
+        for attr_name in Theme.__slots__:
```

Comment:
Since we know in this case it is the `Theme` `__slots__` we want, we do not need to access `_all__slots__`